app/services/square_manager/webhook_validator.rb

```diff
@@ -34,9 +34,17 @@ def call
         )
       )
 
+      apex_signature = Base64.strict_encode64(
+        OpenSSL::HMAC.digest(
+          'sha1',
+          ENV['APEX_WEBHOOK_SIGNATURE_KEY'],
+          string_to_sign
+        )
+      )
```

Comment:
Since this is the third time we're using this, can we abstract this out into a private method?

Reply:
Ofc